src/Microsoft.ML.Data/Transforms/ConversionsExtensionsCatalog.cs

```diff
@@ -143,7 +143,9 @@ public static KeyToValueMappingEstimator MapKeyToValue(this TransformsCatalog.Co
             => new ValueMappingEstimator<TInputType, TOutputType>(CatalogUtils.GetEnvironment(catalog), keys, values, columns);
 
         /// <summary>
-        /// Maps specified keys to specified values
+        /// Maps the <paramref name="columns.input"/> using the keys in the dictionary to the values of dictionary.
+        /// In this case, the <paramref name="lookupMap"/> is used as a dictionary where <paramref name="keyColumn"/>
+        /// and <paramref name="valueColumn"/> specify the keys and values of dictionary respectively.
```

Comment:
Can you add `a value x in the input would be mapped to value stored in dictionary[x]`?